src/components/Footer.js

```diff
@@ -139,171 +140,249 @@ const ComBox = styled.div`
-        list-style: none;
-    };
-
-    a {
-        white-space: nowrap;
-        text-decoration: underline;
-        gap: 20px;
-        color: ${standardColours.white};
+        padding-left: 0;
+        justify-content: space-between;
     };
 `;
 
-const BusinessInfo = styled.div`
+const BusinessInfo = styled.p`
     margin: -16px 0 47px 0;
 
     p {
         color: ${standardColours.white};
     };
 `;
 
+// images
+const StyledLogo = styled(Svg)`
+`;
+
+const StyledIcon = styled(Svg)`
+    height: 32px;
+    width: 32px;
+    cursor: pointer;
+`;
+
+const StyledTrustImg = styled(Svg)`
+    margin-top: 31px;
+`;
+
+const StyledTinyImg = styled(Svg)`
+    margin-right: 8px;
+`;
+
+const StyledNortonImg = styled(Svg)`
+`;
+//
+
 const Footer = () => {
-    // const {
-    //     datoCmsSite: {
-    //         globalSeo: { siteName },
-    //     },
-    //     datoCmsFooter: { boxinfo, boxtitle, companyinfo },
-    //     datoCmsContactInformation: { instagramLink, pinterestLink },
-    //     } = useStaticQuery(graphql`
-    //     query FooterQuery {
-    //         datoCmsSite {
-    //             globalSeo {
-    //             siteName
-    //         }
-    //         }
-    //         datoCmsFooter {
-    //             boxinfo
-    //             boxtitle
-    //             companyinfo
-    //         }
-    //         datoCmsSitewide {
-    //             instagramlink
-    //             pinterestlink
-    //         }
-    //     }
-    //     `);
+    const {
+        datoCmsSite: { siteName },
+        datoCmsFooter: { boxinfo, boxtitle, companyinfo, flagstitle, securitytitle, hovialight },
+        datoCmsSitewide: { instagramprofile, pinterestprofile },
+    } = useStaticQuery(graphql`
+        query FooterQuery {
+            datoCmsSite {
+                globalSeo {
+                    siteName
+                    }
+                }
+            datoCmsFooter {
+                boxinfo
+                boxtitle
+                companyinfo
+                flagstitle
+                securitytitle
+                hovialight {
+                    url
+                    alt
+                    }
+                }
+            datoCmsSitewide {
+                instagramprofile
+                pinterestprofile
+                }
+            }
+        `);
 
     return (
         <StyledFooter>
             <Container>
                 <StyledInner>
                     <StyledTop>
-                        <div className="brand-logo">
-                            <img className="hovia-logo" src={HoviaWhite} alt="hovia"/>
-                        </div>
-                        <StyledSocial>
-                            <img className="insta" src={Insta} alt="Instagram"/>
-                            <img className="pint" src={Pint} alt="Pinterest"/>
-                        </StyledSocial>
+                        <StyledBrand>
+                            <StyledLogo image={hoviaWhite} title={`hovia`}/>
```

Comment:
Can we have this as it's own component within the ui folder so we could use it for the header and footer? Maybe have the default as black and then pass an alt={true} as a prop to switch it to white?